options/base_options.py

```diff
@@ -379,6 +379,20 @@ def initialize(self, parser):
             default=64,
             help="# of discrim filters in the first conv layer",
         )
+        parser.add_argument(
```

Comment:
I believe this is already done with `--D_ndf`.

Reply:
it is removed

Reply:
it is removed